### DISCUSSION TITLE
Allow setting position for all tooltips in a group by definine property on <Tooltip.Group />, rather than on each child

### DISCUSSION THREAD

**Discussion by Nantris** [mantinedev/mantine#2194] [Feature requests]
This would reduce duplicated code and ease the upgrade path from other tooltip libraries.

**Comment by rtivital** [comment]
No, we will not do that, we've had similar logic in Radio.Group and other group components, it caused a lot of confusion. It is better to separate props – Tooltip.Group is responsible for grouping behavior, Tooltip is responsible for everything else.
